tests/cypress/pageAction/customizeStore.action.js

```diff
@@ -0,0 +1,64 @@
+import BluehostHomePage from '../pageObject/bluehostHome.page';
+import CustomizeStorePage from '../pageObject/dashboard/customizeYourStore.page';
+
+export function verifyCard( cards ) {
+	BluehostHomePage.customizeYourTab().click();
+	cards.forEach( ( element ) => {
+		cy.contains( element );
+	} );
+}
+
+export function createPages() {
+	createHomePage();
+	createAboutPage();
+	createContactPage();
+}
+
+function createHomePage() {
+	BluehostHomePage.customizeYourTab().click();
+	CustomizeStorePage.homePage().click();
+	CustomizeStorePage.yithThemeHomePageHeader().should( 'exist' );
+	CustomizeStorePage.waitForLoaderTill( 10000 );
+	CustomizeStorePage.publishButton().click( { force: true } );
+	CustomizeStorePage.editorPostPublishButton().click( { force: true } );
```

Comment:
We should avoid the use of `{ force: true }` at all costs. This can hide a number of potential UI issues that would otherwise be detected. I'd recommend doing a search for all instances of this and removing them.